statsmodels/genmod/families/family.py

```diff
@@ -517,17 +523,27 @@ def loglike(self, endog, mu, scale=1.):
         llf = sum((`endog`*`mu`-`mu`**2/2)/`scale` - `endog`**2/(2*`scale`) - \
             (1/2.)*log(2*pi*`scale`))
         """
+
+        va = scale * self.variance(mu)
```

Comment:
I think `scale` should be the same as `self.variance(mu)` already in the common, regular use case, but we should be able to fix scale independently of variance(mu)

I'm not sure yet